admin_guide/manage_scc.adoc

```diff
@@ -277,6 +284,10 @@ Then, add the service account to the `privileged` SCC.
 $ oadm policy add-scc-to-user privileged system:serviceaccount:myproject:mysvcacct
 ----
 
+Then, ensure that the resource is being created on behalf of the service
+account. To do so, set the `spec.serviceAccountName` field to a service account
```

Comment:
Leaving the service account name blank will result in the `default` service account being used.